app/domain/finders/content.rb

```diff
@@ -22,42 +22,75 @@ def call
   ALL = 'all'.freeze
   NONE = 'none'.freeze
 
-  attr_reader :organisation_id, :document_type, :date_range, :search_term
+  attr_reader :filter
 
   def initialize(filter)
-    @organisation_id = filter.fetch(:organisation_id)
-    @document_type = filter.fetch(:document_type)
-    @search_term = parse_search_term(filter[:search_term]) if filter[:search_term]
-    @page = filter[:page] || 1
-    @page_size = filter[:page_size] || DEFAULT_PAGE_SIZE
+    @filter = filter
+
+    @page = filter.fetch(:page, 1)
+    @page_size = filter.fetch(:page_size, DEFAULT_PAGE_SIZE)
     @date_range = filter.fetch(:date_range)
     @sort_attribute = filter.fetch(:sort_attribute) || 'upviews'
     @sort_direction = filter.fetch(:sort_direction) || 'desc'
```

Comment:
Instead of following the same pattern, I am reverting my initial refactor.  This is why: 

The default attribute for `fetch` (the second argument) only applies when the `key` is not present, not when it is `nil`. In our tests we set default values to `nil` so they are failing if I am consistent with the changes to `page` and `page_size`. 

The purpose of this PR is not to refactor this class, so I will keep the scope small by leaving it in its initial state. Thank for your raising this issue.